integration-testing/examples/jvm-sample/build.gradle.kts

```diff
@@ -1,10 +1,14 @@
 buildscript {
     repositories {
         mavenLocal()
```

Comment:
It seems to me that in the future it is better to use custom repositories for functional tests, rather than `mavenLocal()`

Reply:
+ I've added that to the ticket: https://github.com/Kotlin/kotlinx-atomicfu/issues/362